tools/miniasm/miniasm.xml

```diff
@@ -0,0 +1,81 @@
+                <param argument="minIden" type="float" value="0.05" label="During read filtering, ignore mappings with col10/col11 below this value"
+                       help="Ignored mappings are still used for read overlaps" />
+                <param argument="minSpan" type="integer" value="1000" label="Drop mappings shorter than this number bp"
+                       help="This option also affects the second round of read filtering and minimal overlap length"/>
+                <param argument="minCov" type="integer" value="3" label="Minimal coverage by other reads"
+                       help="In the first round of filtering, miniasm finds the longest region covered by this number or more reads. In the second round, it in addition requires each remaining base to be covered by  INT  bases  at  least minSpan/2 from the ends of other reads" />
+                <param argument="minOvlp" type="integer" value="1000" label="Minimal overlap length [same as minSpan]" />
+                <param argument="maxHang" type="integer" value="1000" label="Maximum overhang length"
+                       help="An overhang is an unmapped region that should be mapped given a true overlap or true containment. If the overhang is too long, the mapping is considered an internal match and will be ignored" />
+                <param argument="intThres" type="float" value="0.08" label="Minimal ratio of mapping length to mapping+overhang length for a mapping considered a containment or an overlap"
+                       help="This option has a similar role to MaxHang, except that it controls the ratio, not length" />
+                <param argument="maxGapDiff" type="integer" value="1000" label="Maximal gap differences between two reads in a mapping"
+                       help="This parameter is only used for transitive reduction" />
+                <param argument="maxDubDist" type="integer" value="50000" label="Maximal probing distance for bubble popping"
+                       help="Bubbles longer than this value will not be popped" />
+                <param argument="minUtgSize" type="integer" value="4" label="A unitig is considered small if it is composed of less than this many reads"
+                       help="Miniasm may try to remove small unitigs at various steps"/>
+                <param argument="nRounds" type="integer" value="3" label="Rounds of short overlap removal" help=""/>
+                <param argument="finalDropRatio" type="float" value="0.8" label="Overlap drop ratio threshold after short unitig removal" help=""/>
+            </when>
+        </conditional>
+    </inputs>
+    <outputs>
+        <data name="gfa" format="tabular" label="${tool.name} on ${on_string}: Assembly Graph" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="readFile" value="mini_reads.fq"/>
+            <param name="paf" value="mini_reads.paf.gz"/>
+            <output name="gfa" file="mini_reads.gfa"/>
```

Comment:
kun je nog een test maken waarbij ook wat van de advanced parameters gebruikt worden?

Reply:
spoilers: dat gaat nu mis, je moet dan `$settings.MinMatch` etc gebruiken als parameter naam in je `command` omdat deze binnen de conditional staan. Verder kun je de conditional misschien ook vervangen door een `section`, dat is wat simpeler (zie mijn cheat sheet voor een voorbeeld)

Reply:
btw, aangezien ik wat commits gepusht heb naar jouw branch, voor je zometeen nog meer aanpassingen maakt, eerste een `git pull` doen :)